lib/hanami/model/plugins/timestamps.rb

```diff
@@ -24,7 +24,7 @@ class InputWithTimestamp < WrappingInput
           def initialize(relation, input)
             super
             columns     = relation.columns.sort
```

Comment:
Previously when it was `== TIMESTAMPS` it was important. There's no need anymore. 
You could also remove the temp var
`@timestamps = relation.columns & TIMESTAMPS`